src/helper.rs

```diff
@@ -12,12 +12,12 @@ const DEFAULT_TEMPLATE: &'static str = "README.tpl";
 /// The given path is appended to the current directory if is a relative path, otherwise it is used
 /// as is. If no path is given, the current directory is used.
 /// A `Cargo.toml` file must be present is the root directory.
-pub fn get_project_root(given_root: Option<&str>) -> Result<PathBuf, String> {
+pub fn get_project_root(given_root: Option<&String>) -> Result<PathBuf, String> {
```

Comment:
This seems a little odd. I wonder if best written as e.g.

```suggestion
pub fn get_project_root<S: AsRef<str>>(given_root: Option<S>) -> Result<PathBuf, String> {
```

Reply:
I'll take a look, I didn't even realize these are pub. What I should do is leave the signature as-is and do my own conversion to use the function.

Reply:
btw Option<T> is an annoying interface because you have to type-specialize when passing None